tests/test_big.py

```diff
@@ -0,0 +1,42 @@
+# SPDX-FileCopyrightText: 2023 Henrik Sandklef
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import json
+import pytest
+
+from flict.flictlib.arbiter import Arbiter
+
+arbiter = Arbiter()
+
+def _test_expression(inbound, outbound, result):
+    compats = arbiter.inbounds_outbound_check(outbound, [inbound])
+    assert result == compats['compatibility']
+
+def test_big_or():
```

Comment:
I would recommend to comment on what the goal of this test is or what scenario this is trying to recreate as it is just a giant combination of licenses.  
  
Is there a reason this is so long?

Reply:
This bug (causing this fix and PR) was found in a slightly complex license expression with a "WITH" statement. The idea with a really long license is to make sure a parser error is found. 

And it doesn't take that long time to run.

Adding comments in the code.